### PR TITLE
feat: home page

### LINKED ISSUE #427: Add home page frontend

### PR DESCRIPTION
**Pull request by ruilopesm**
TODO:
- [x] Infinite scroll (@ruilopesm)
- [x] Make the schedule sticky (@joaodiaslobo)
- [x] Schedule on mobile (@ruilopesm)
- [x] Following tab (@danielsp45)
- [x] Watch out for empty states on the schedule (@danielsp45)
- [x] Create more components, leading to a more modular code (@ruilopesm)
- [x] Normalize all index pages styling (@ruilopesm)

### CODE REVIEW

**Comment by diogogmatos** [comment]
[Home Page](https://trello.com/c/GFItbprE/6-home-page)

File: lib/atomic/feed.ex [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+      iex> list_posts()
+      %{entries: [%Post{}], metadata: %{...}}
+
+  """
+  def list_posts(opts \\ []) do
+    Post
+    |> apply_filters(opts)
+    |> preload(activity: :organization, announcement: :organization)
+    |> Repo.paginate(cursor_fields: [:inserted_at, :id], limit: 50)
```
Comment (ruilopesm) [inline lib/atomic/feed.ex]: Make `limit` a module attribute, in order to reuse it in other functions

File: lib/atomic/feed.ex [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    Repo.delete(post)
+  end
+
+  @doc """
+  Returns a data structure for tracking post changes.
+
+  ## Examples
+
+      iex> change_post(post)
+      %Todo{...}
```
Comment (ruilopesm) [inline lib/atomic/feed.ex]:
```suggestion
      %Post{...}
```

File: lib/atomic/organizations.ex [hunk 3]
```diff
@@ -34,6 +35,48 @@ defmodule Atomic.Organizations do
...
+  @doc """
+  Returns a list of organizations that are not followed by an user, ordered by the amount of followers.
+
+  ## Examples
+
+      iex> list_top_organizations_by_user(user)
+      [%Organization{}, ...]
+
+  """
+  def list_top_organizations_by_user(%User{} = user, opts \\ []) do
```
Comment (ruilopesm) [inline lib/atomic/organizations.ex]:
```suggestion
  def list_top_organizations_for_user(%User{} = user, opts \\ []) do
```

File: lib/atomic/activities.ex [hunk 4]
```diff
@@ -176,21 +177,45 @@ defmodule Atomic.Activities do
...
       {:ok, %Activity{}}
 
+      iex> create_activity(%{field: bad_value, ~N[2020-01-01 00:00:00]})
+      {:error, %Ecto.Changeset{}}
+
       iex> create_activity(%{field: bad_value})
       {:error, %Ecto.Changeset{}}
 
   """
   def create_activity(attrs \\ %{}) do
```
Comment (ruilopesm) [inline lib/atomic/activities.ex]:
```suggestion
  def create_activity_with_post(attrs \\ %{}) do
```
And create a simpler function called `create_activity`.
(Same for announcements)

File: lib/atomic/feed.ex [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+  Returns a cursor-based pagination of posts.
+  Learn more about why using cursor-based pagination at: https://use-the-index-luke.com/no-offset
+
+  ## Examples
+
+      iex> list_posts()
+      %{entries: [%Post{}], metadata: %{...}}
+
+  """
+  def list_posts(opts \\ []) do
```
Comment (ruilopesm) [inline lib/atomic/feed.ex]:
```suggestion
  def list_paginated_posts(opts \\ []) do
```
And create a simpler function called `list_posts`.
(Maybe there's a way to fuse this with the "next" one, by using named parameters)

**Review by danielsp45 — APPROVED** [review]
LESSGOO 🚀 ⚛️

**Review by joaodiaslobo — APPROVED** [review]
### **OMG LETS GO** 🚀🚀🚀🚀🚀🚀

**Review by MarioRodrigues10 — COMMENTED** [review]
LETSSSS GOOOO :rocket:

**Comment by ferjpires** [comment]
LETS GOOOOOOO

**Pull request merged**